### PR TITLE
feat(tele): Add amazonq_ (awqs_) metrics

### PR DESCRIPTION
**Pull request by dianaabv16**
## Problem
- currently the name of telemetry metrics for weaverbird is `awsq_` it should be `amazonq_`
- we are changing the format of out metrics.  we want to able to have more centralised metrics.

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: telemetry/definitions/commonDefinitions.json [hunk 1]
```diff
@@ -2663,6 +2704,89 @@
             "description": "User has accepted the approach generated by Weaverbird.",
             "metadata": [{ "type": "enabled" }, { "type": "awsqConversationId" }] 
         },
+        {
```
Comment (jpinkney-aws) [inline telemetry/definitions/commonDefinitions.json:2707]: I see that a few things are added but none are deleted, are we going to remove all the awsq ones that are currently being used here or is that a next step?

Comment (dianaabv16) [reply telemetry/definitions/commonDefinitions.json:2707]: I think to make it backwards compatible.
I think it is better to keep awsq_ metrics until we are fully moved to aamazonq_
what do you think>

Comment (jpinkney-aws) [reply telemetry/definitions/commonDefinitions.json:2707]: I think that's fine so we don't block other teams but we have to have a task to make sure to remove them

Comment (dianaabv16) [reply telemetry/definitions/commonDefinitions.json:2707]: I will add one

Comment (dhruvigajjar) [reply telemetry/definitions/commonDefinitions.json:2707]: Is the change live?

**Review by jpinkney-aws — APPROVED** [review]

File: telemetry/definitions/commonDefinitions.json (hunk 1 above)
Comment (dianaabv16) [reply telemetry/definitions/commonDefinitions.json:2707]: no, I this is to register new events.I will open second CR in aws-toolkit-vscode-staging

Comment (jpinkney-aws) [reply telemetry/definitions/commonDefinitions.json:2707]: Yes, we've had awsq metrics for ~1.5 weeks

**Pull request merged**